### DISCUSSION TITLE
context menue of side bare should include "open in terminal", "open as root" and be dropable to the terminal

### DISCUSSION THREAD

**Discussion by olisnr** [linuxmint/discussions#626] [Ideas]
i love nemo, and would like, if the left folder/places sidbar, and also the click-able adress-line would support thinks that also the main-window allows:
open as root, drag and drop an path into a terminal, and so on.

is this easy possible?

**Comment by Jopp-gh** [comment]
I don't fully understand your concern but it seems you are looking for **Nemo actions**. Check out [this](https://github.com/smurphos/nemo_actions_and_cinnamon_scripts) Nemo actions and tweak them to your liking.

**Comment by olisnr** [comment]
@Jopp-gh i tried to show it in a clip. i would like to have the same "terminal" and "root" context-menu items anywhere...


https://github.com/user-attachments/assets/47c7aa68-9585-4263-8939-ae5c97b7c5dd
